src/aiida_quantumespresso/workflows/ph/base.py

```diff
@@ -22,8 +22,9 @@ class PhBaseWorkChain(ProtocolMixin, BaseRestartWorkChain):
 
     defaults = AttributeDict({
         'delta_factor_max_seconds': 0.95,
-        'delta_factor_alpha_mix': 0.90,
-        'alpha_mix': 0.70,
+        'delta_factor_alpha_mix': 0.5,
+        'nmix_ph': 4,
+        'alpha_mix': 0.4,
```

Comment:
I think we want to set this as the protocol default, no? So we should add this to the `protocols/ph/base.yaml` file instead.